contracts/sweep/TokenSweep.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+/**
+ * @title Abstract upgradeable contract that provides the ability to sweep erroneous transferred tokens to a defined beneficiary address
+ *
+ * @dev Access control implementation is required for many functions by design
+ */
+abstract contract TokenSweep {
+    address internal _beneficiary;
+
+    event BeneficiaryUpdated(address indexed beneficiary);
```

Comment:
issue: tense of the event needs to be consistent with the other contracts.

Elsewhere events are in present tense, rather than the past e.g. `BeneficiaryUpdate` vs `BeneficiaryUpdated`

Don't really mind which tense we use, but we want to keep it consistent ...if you want to switch, it's a broad update across all contracts 😉

Reply:
past tense it is! makes more sense because the its already happened by the time you see the log